yaml_files/simphony_metadata.yml

```diff
@@ -454,22 +454,24 @@ CUDS_KEYS:
 
   DIRICHLET:
     definition: Dirichlet boundary condition to specify the value the solutions takes on the boundary of the domain.
-    parent: DIFFERENTIAL_EQUATION_CONDITION
+    parent: FIELD_CONDITION
 
   NEUMANN:
     definition: Neumann boundary condition, it specifies the values that the derivative of a solution with respect to the variable is to take on the boundary of the domain.
-    parent: DIFFERENTIAL_EQUATION_CONDITION
+    parent: FIELD_CONDITION
 
   ROBIN:
-    parent: DIFFERENTIAL_EQUATION_CONDITION
+    parent: FIELD_CONDITION
     definition: A mixed boundary condition $\alpha \Phi (x) + \beta (x) \partial {\Phi} / \partial {\bf{n}} (x) = h(x)$, with $h$ is the value (CUBA.VALUE).
     CUBA.DIRECHLET:
     CUBA.NEUMANN:
 
   EMPTY:
-    definition: Empty boundary condition
-    parent: CUBA.VALUE_CONDITION
-    value: CUBA.NONE
+    definition: an entity to represent that no boundary condition is applied on that domain
+    parent: CUBA.CONDITION
+    CUBA.VALUE:
+      default: CUBA.NONE
```

Comment:
This is not consistent. CUBA.VALUE is not defined anywhere. It is not in cuba.yml. It is not in simphony_metadata.yml. 

If it were in cuba.yml, it would be a datatype and must therefore be int, string, double. None is not an option, first because we can't have a datatype of type None (it would not accept _anything_ else except None, which makes it quite useless)

if it were in simphony_metadata, then it would be a CUDS item, and by design the property must match the type, hence CUBA.NONE must be a subclass of CUBA.VALUE.